Detectors/TOF/calibration/include/TOFCalibration/CalibTOFapi.h

```diff
@@ -0,0 +1,68 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file CalibTOFapi.h
+/// \brief Class to use TOF calibration (decalibration, calibration)
+/// \author Francesco.Noferini@cern.ch, Chiara.Zampolli@cern.ch
+
+#ifndef ALICEO2_TOF_CALIBTOFAPI_H_
+#define ALICEO2_TOF_CALIBTOFAPI_H_
+
+#include <iostream>
+#include "CCDB/BasicCCDBManager.h"
+#include "DataFormatsTOF/CalibLHCphaseTOF.h"
+#include "DataFormatsTOF/CalibTimeSlewingParamTOF.h"
+
+namespace o2
+{
+namespace tof
+{
+
+class CalibTOFapi
+{
+
+  using lhcPhase = o2::dataformats::CalibLHCphaseTOF;
+  using slewParam = o2::dataformats::CalibTimeSlewingParamTOF;
+  using ccdbManager = o2::ccdb::BasicCCDBManager;
```

Comment:
types declared with `using` should be capitalized like any other type

Reply:
done